src/cascadia/TerminalSettingsModel/CascadiaSettings.cpp

```diff
@@ -202,6 +202,36 @@ winrt::Microsoft::Terminal::Settings::Model::GlobalAppSettings CascadiaSettings:
     return *_globals;
 }
 
+// Method Description:
+// - Get a reference to our default profile settings
+// Arguments:
+// - <none>
+// Return Value:
+// - a reference to our default profile settings
+winrt::Microsoft::Terminal::Settings::Model::Profile CascadiaSettings::DefaultProfileSettings() const
+{
+    return *_userDefaultProfileSettings;
+}
+
+// Method Description:
+// - Create a new profile based off the default profile settings.
+// Arguments:
+// - <none>
+// Return Value:
+// - a reference to the new profile
+winrt::Microsoft::Terminal::Settings::Model::Profile CascadiaSettings::CreateNewProfile()
+{
+    auto newProfile{ _userDefaultProfileSettings->CreateChild() };
+    _allProfiles.Append(*newProfile);
+
+    // Give the new profile a distinct name so a guid is properly generated
+    auto newName = fmt::format(L"Profile {}", _allProfiles.Size());

```

Comment:
```suggestion
    const auto newName = fmt::format(L"Profile {}", _allProfiles.Size()+1);
```
The smallest nit ever. But this way, the 1st profile will be called "Profile 1" not "Profile 0".

Reply:
It actually turns out you're wrong :wink: and I was wrong for the same reason.

He asks for Size() _AFTER INSERTING THE NEW PROFILE_

Reply:
🤣 gotcha both there

Reply:
😲